### ISSUE TITLE
Convert Existing Files for Compatability with phpVMS

### ISSUE THREAD

**Issue by HenrikBerg02** [IFAO/Website#7]
I believe this would just be some small tweaking to the code, so it shouldn't be a big issue.

**Comment by BVRAU** [comment]
Yes, it is only adding the php in amongst what is already there

**Comment by HenrikBerg02** [comment]
PHP can be integrated within HTML code, correct?

**Comment by BVRAU** [comment]
Yes, think of PHP as the code in your computer OS, and the HTML as the GUI
